src/rules/no-standalone-expect.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import {
+  createRule,
+  isDescribe,
+  isExpectCall,
+  isFunction,
+  isTestCase,
+} from './tsUtils';
+
+const getBlockType = (
+  stmt: TSESTree.BlockStatement,
+): 'function' | 'describe' | 'callExpr' | false => {
```

Comment:
I think I'd prefer returning `null` instead of `false`; in particular, TS treats `null` & `undefined` a bit differently that can be useful vs `false`.

Also would be nice to have this as type alias :)